OpenRA.Game/Network/SyncReport.cs

```diff
@@ -92,7 +92,8 @@ void GenerateSyncReport(Report report)
 
 		internal void DumpSyncReport(int frame, IEnumerable<FrameData.ClientOrder> orders)
 		{
-			Log.AddChannel("sync", "syncreport.log");
+			var reportName = "syncreport-" + DateTime.UtcNow.ToString("yyyy-MM-ddTHHmmssZ") + ".log";
```

Comment:
Consider using [`ToString("u")`](https://msdn.microsoft.com/en-us/library/az4se3k1(v=vs.110).aspx#UniversalSortable) which is a touch more human readable, but still sortable.